main.py

```diff
@@ -0,0 +1,53 @@
+import math
+import csv
+import requests
+from bs4 import BeautifulSoup
+import sys
+
+
+def get_distance(loc1, loc2):
+    return round(math.sqrt((loc1[0] - loc2[0]) * (loc1[0] - loc2[0]) + (loc1[1] - loc2[1]) * (loc1[1] - loc2[1])), 4)
```

Comment:
https://stackoverflow.com/questions/936541/math-pow-vs-multiply-operator-performance

This is my reasoning

Reply:
I got used to avoid the math.pow, the alternative supporting performance, even though it is not that clean:
https://stackoverflow.com/questions/936541/math-pow-vs-multiply-operator-performance